VideoLocker/res/values/styles.xml

```diff
@@ -106,6 +105,10 @@
         <item name="android:background">@color/edx_brand_primary_base</item>
     </style>
 
+    <style name="course_dashboard_separator_style" parent="separator_style">
```

Comment:
Find out how many separator colors are used throughout the app, and if they are standardized in the style guidelines ;) Then we can decide the naming scheme based on that.
> On 23-Sep-2015, at 12:17 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/values/styles.xml <https://github.com/edx/edx-app-android/pull/368#discussion_r40176074>:
> 
> > @@ -106,6 +105,10 @@
> >          <item name="android:background">@color/edx_brand_primary_base</item>
> >      </style>
> >  
> > +    <style name="course_dashboard_separator_style" parent="separator_style">
> so whats the final verdict ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/368/files#r40176074>.
>